ansible/files/admin_api_scripts/commence_walg_restore.sh

```diff
@@ -0,0 +1,39 @@
+#! /usr/bin/env bash
+
+function commence_walg_restore {
+    backup_name=$1
+    recovery_target_time=$2
+
+    echo $recovery_target_time
+    
+    # Stop database and empty it
+    systemctl stop postgresql
+    rm -rf /var/lib/postgresql/data/*
+
+    # Download base backup
+    wal-g backup-fetch /var/lib/postgresql/data $backup_name --config /etc/wal-g/config.json
```

Comment:
quote $backup_name (consider running shellcheck on these scripts to catch these issues)

Personally I like to enable https://www.shellcheck.net/wiki/SC2250 as well but that's more of a debatable preference.